samples/dotnet/kernel-syntax-examples/Resources/EmbeddedResource.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Reflection;
+using RepoUtils;
+
+namespace Resources;
+
+internal static class EmbeddedResource
+{
+    private static readonly string? s_namespace = typeof(EmbeddedResource).Namespace;
```

Comment:
Nit:  I don't think this can ever be null so you don't need to but the '?' after "string".  If the editors give you a hard time for doing that, you can then put a '!' after "Namespace" to get rid of warnings.

Reply:
it's a default implementation I copy&paste where needed, it allows scenarios where it can be null. I rather not customize the code for each scenario